examples/src/utils/PointCloudClassificationFilterUI.ts

```diff
@@ -16,12 +16,12 @@ export class PointCloudClassificationFilterUI {
   }
 }
 
-function getClassName(clazz: number): string {
-  const entry = Object.entries(WellKnownAsprsPointClassCodes).find(entry => {
-    if (entry[1] === clazz) {
-      return true;
-    }
-    return false;
-  })
+function getClassName(clazz: number | string): string {
+  if (typeof(clazz) === 'string') {
```

Comment:
Why it's called "clazz", not "class"?

Reply:
I believe it's because `class` is a reserved keyword

Reply:
Alright, I'll see if we can rename it to something else.. I just want to make sure it's obvious it's not a spelling mistake :sweat_smile: